### PR TITLE
new keyword field for formbuilder

### PR DESCRIPTION
**Pull request by hbaptiste**
new keyword field for formbuilder

### CODE REVIEW

**Comment by mickaelandrieu** [comment]
Thank you @hbaptiste.

@ndufreche would you mind to review?  :)

File: src/tb/component/autocomplete/main.js [hunk 1]
```diff
@@ -0,0 +1,212 @@
...
+                idKey: "uid"
+            });
+            this.suggestionList.onInit = this.updateUi.bind(this);
+            this.handleDataSource();
+            this.threshold = parseInt(this.settings.threshold, 10) || 3;
+            this.lastRequest = null;
+            this.currentSelection = null;
+            this.field = jQuery(this.settings.field);
+            this.isPending = false;
+            jQuery(this.field).css('position', 'relative');
```
Comment (ndufreche) [inline src/tb/component/autocomplete/main.js]: this can't be in less file directly ?

Comment (hbaptiste) [inline src/tb/component/autocomplete/main.js]: Not worth it I think.

Comment (ndufreche) [inline src/tb/component/autocomplete/main.js]: ok.

**Comment by ndufreche** [comment]
:+1: thanks

**Pull request merged**